### PR TITLE
Use only one network for each test class

### PR DESCRIPTION
**Pull request by flomillot**
Signed-off-by: Florent MILLOT <millotflo@gmail.com>

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java [hunk 1]
```diff
@@ -61,17 +59,15 @@
 @AutoConfigureMockMvc
 public abstract class AbstractNetworkModificationTest {
 
-    protected static final UUID TEST_NETWORK_ID = UUID.randomUUID();
-    protected static final UUID NOT_FOUND_NETWORK_ID = UUID.randomUUID();
+    private static final UUID TEST_NETWORK_ID = UUID.randomUUID();
+    private static final UUID NOT_FOUND_NETWORK_ID = UUID.randomUUID();
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java:63]: Make private `TEST_GROUP_ID`

Comment (flomillot) [reply src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java:63]: I can't it's used

Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java:63]: add a protected method getGroupUuid ?

**Review by SlimaneAmar — APPROVED** [review]

File: src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java (hunk 1 above)
Comment (flomillot) [reply src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java:63]: Done

**Pull request merged**